src/main/scala/mockws/MockWS.scala

```diff
@@ -31,7 +33,7 @@ import play.api.mvc.EssentialAction
  *
  * @param routes routes defining the mock calls
  */
-class MockWS(routes: MockWS.Routes, shutdownHook: () ⇒ Unit)(implicit val materializer: ActorMaterializer) extends WSClient {
+class MockWS(routes: MockWS.Routes, shutdownHook: () ⇒ Unit)(implicit val materializer: ActorMaterializer, action: RouteNotDefined) extends WSClient {
```

Comment:
I like the name `notFoundBehaviour` you used before.
Could you use it here instead of `action`?

Reply:
here Here 2d9bb9f02a897ea69f22bdb30d20831046c120e7